app/src/main/java/org/schabi/newpipe/fragments/list/search/SearchFragment.java

```diff
@@ -250,13 +269,6 @@ public void onResume() {
         }
         super.onResume();
 
-        try {
-            service = NewPipe.getService(serviceId);
-        } catch (final Exception e) {
-            ErrorActivity.reportUiErrorInSnackbar(this,
-                    "Getting service for id " + serviceId, e);
-        }
-
```

Comment:
I don't understand this. onResume() is called when you change service? then it should be updated onResume()? anyway search & switching service works

Reply:
@B0pol Did you figure something out here to merge this? It seems you had a question that wasn't resolved 🤔

Reply:
> anyway search & switching service works